addons/base_vat/models/res_partner.py

```diff
@@ -821,10 +821,16 @@ def create(self, vals_list):
             if values.get('vat'):
                 country_id = values.get('country_id')
                 values['vat'] = self._fix_vat_number(values['vat'], country_id)
-        return super(ResPartner, self).create(vals_list)
+        res = super().create(vals_list)
+        if self.env.context.get('import_file'):
+            res.env.remove_to_compute(self._fields['vies_valid'], self)
```

Comment:
I think it should be:
```suggestion
            res.env.remove_to_compute(self._fields['vies_valid'], res)
```